runner.yml.dist

```diff
@@ -28,7 +28,9 @@ drupal:
     - "./vendor/bin/drush en oe_theme_content_project -y"
     - "./vendor/bin/drush en oe_theme_content_call_tenders -y"
     - "./vendor/bin/drush en oe_theme_content_call_proposals -y"
+    - "./vendor/bin/drush en oe_theme_content_consultation -y"
     - "./vendor/bin/drush en oe_theme_content_organisation -y"
+    - "./vendor/bin/drush en oe_theme_content_document_reference -y"
```

Comment:
Might be oe_theme_content_consultation has to have as a dependency? In this case we don't need to enable oe_theme_content_document_reference here.